content/substrate/intro/slides.md

```diff
@@ -616,7 +616,7 @@ Notes:
 - Can be slower than native, depending on the executor/execution method.
 - Limited access to the host host env, all needs to be done through syscalls.
 
-Less state-transition diversification, because the runtime is the same for all clients. If there a
+Less state-transition diversification, because the runtime is the same for all clients. If there is a
 bug in it, everyone is affected.
```

Comment:
```suggestion
Less state-transition diversification, because the runtime is the same for all clients.
If there is a bug in it, everyone is affected.
```